### PR TITLE
remove unused image informer

### PR DESCRIPTION
**Pull request by juanvallejo**
cc @deads2k @ericavonb @stlaz

### CODE REVIEW

**Comment by ericavonb** [comment]
I don't have much to say on this one. Perhaps @bparees will want to take a look?

File: pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go [hunk 1]
```diff
@@ -220,7 +220,7 @@ type KubeAPIServerInformers struct {
 	InternalKubernetesInformers        kinternalinformers.SharedInformerFactory
 	KubernetesInformers                kexternalinformers.SharedInformerFactory
 	OpenshiftOAuthInformers            oauthinformer.SharedInformerFactory
-	InternalOpenshiftImageInformers    imageinformer.SharedInformerFactory
+	InternalOpenshiftImageInformers    imagev1informer.SharedInformerFactory
```
Comment (bparees) [inline pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go]: should this field be renamed if it's not an internal informer anymore?

File: pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go [hunk 2]
```diff
@@ -232,7 +232,7 @@ func (i *KubeAPIServerInformers) GetInternalKubernetesInformers() kinternalinfor
 func (i *KubeAPIServerInformers) GetKubernetesInformers() kexternalinformers.SharedInformerFactory {
 	return i.KubernetesInformers
 }
-func (i *KubeAPIServerInformers) GetInternalOpenshiftImageInformers() imageinformer.SharedInformerFactory {
+func (i *KubeAPIServerInformers) GetInternalOpenshiftImageInformers() imagev1informer.SharedInformerFactory {
```
Comment (bparees) [inline pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go]: and this function?

**Comment by bparees** [comment]
what's motivating this change and why are the other informers not changing?

**Comment by juanvallejo** [comment]
> what's motivating this change and why are the other informers not changing?

Upstream changes, such as https://github.com/kubernetes/kubernetes/pull/69685, are moving to external informers and listers. I have two other pulls updating informers for [security](https://github.com/openshift/origin/pull/21391) and [authorization](https://github.com/openshift/origin/pull/21408). We're holding off on updating quota informers for now.

cc @deads2k

File: pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go (hunk 1 above)
Comment (juanvallejo) [reply pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go]: Thanks, done

File: pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go (hunk 2 above)
Comment (juanvallejo) [reply pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go]: Thanks, fixed

File: pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go [hunk 3]
```diff
@@ -232,8 +232,8 @@ func (i *KubeAPIServerInformers) GetInternalKubernetesInformers() kinternalinfor
 func (i *KubeAPIServerInformers) GetKubernetesInformers() kexternalinformers.SharedInformerFactory {
 	return i.KubernetesInformers
 }
-func (i *KubeAPIServerInformers) GetInternalOpenshiftImageInformers() imageinformer.SharedInformerFactory {
-	return i.InternalOpenshiftImageInformers
+func (i *KubeAPIServerInformers) GetOpenshiftImageInformers() imagev1informer.SharedInformerFactory {
```
Comment (deads2k) [inline pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go]: wait, if this renamed worked, who is using this thing?

Comment (juanvallejo) [reply pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go]: I am unable to find any users of it prior to this change

**Comment by juanvallejo** [comment]
@deads2k @bparees  friendly ping

**Comment by bparees** [comment]
i don't have a problem w/ the change other than being confused why we have this if no one is using it.

**Comment by juanvallejo** [comment]
@bparees @deads2k thanks, removed unused internal image informer pieces altogether

**Pull request merged**